requirements.txt

```diff
@@ -7,14 +7,12 @@ canonicalwebteam.yaml-responses[django]==1.1.0
 canonicalwebteam.get-feeds==0.2.4
 canonicalwebteam.http==0.1.6
 whitenoise==4.1.2
-gunicorn[gevent]
 statsd==3.3.0
 prometheus_client==0.6.0
-talisker==0.14.2
+talisker[gunicorn]==0.14.2
```

Comment:
This should now be `0.14.3` I think. @tbille could confirm.

Reply:
Updated it. Its the latest version